.github/workflows/test-release.yml

```diff
@@ -23,6 +23,10 @@ jobs:
           - "2.7"
           - "3.1"
           - "3.2"
+          - "3.3"
+        activerecord:
+          - "7.1"
+          - "7.2"
     steps:
       - uses: actions/checkout@v3
```

Comment:
can you rev actions/checkout to v4?

Reply:
sure!

Reply:
done in: 82a9d4506a6e12c123a427e3f87fe796587e24a0